packages/trpc/server/routers/viewer/slots/util.ts

```diff
@@ -483,158 +488,50 @@ async function _getAvailableSlots({ input, ctx }: GetScheduleOptions): Promise<I
   const skipContactOwner = input.skipContactOwner;
   const contactOwnerEmail = skipContactOwner ? null : contactOwnerEmailFromInput;
 
-  let routedHostsWithContactOwnerAndFixedHosts = getRoutedHostsWithContactOwnerAndFixedHosts({
+  const routedHostsWithContactOwnerAndFixedHosts = getRoutedHostsWithContactOwnerAndFixedHosts({
     hosts: eventHosts,
     routedTeamMemberIds: input.routedTeamMemberIds ?? null,
     contactOwnerEmail,
   });
 
-  if (
```

Comment:
We move all this logic into a separate function. Since we need to calculate the availability of the contact owner. If the first slot is more than 2 weeks away we then need to calculate the availability of the other members.